crates/egui-winit/src/lib.rs

```diff
@@ -405,14 +387,20 @@ impl State {
                 }
             }
             WindowEvent::ModifiersChanged(state) => {
-                self.egui_input.modifiers.alt = state.alt();
-                self.egui_input.modifiers.ctrl = state.ctrl();
-                self.egui_input.modifiers.shift = state.shift();
-                self.egui_input.modifiers.mac_cmd = cfg!(target_os = "macos") && state.logo();
+                use winit::keyboard::ModifiersKeyState::Pressed;
+                self.egui_input.modifiers.alt =
+                    state.lalt_state() == Pressed || state.ralt_state() == Pressed;
+                self.egui_input.modifiers.ctrl =
+                    state.lcontrol_state() == Pressed || state.rcontrol_state() == Pressed;
+                self.egui_input.modifiers.shift =
+                    state.lshift_state() == Pressed || state.rshift_state() == Pressed;
+                self.egui_input.modifiers.mac_cmd = cfg!(target_os = "macos")
+                    && (state.lsuper_state() == Pressed || state.rsuper_state() == Pressed);
+
                 self.egui_input.modifiers.command = if cfg!(target_os = "macos") {
-                    state.logo()
+                    state.lsuper_state() == Pressed || state.rsuper_state() == Pressed
                 } else {
-                    state.ctrl()
+                    state.lcontrol_state() == Pressed || state.rcontrol_state() == Pressed
```

Comment:
Maybe this is really what's intended here but it does seem kinda odd to me that egui_input specifically exposes a "mac_cmd" modifier as the super key clamped to false if not macOS, and then again exposes the same modifier via `modifiers.command` except mapped to `ctrl` if not on macOS.

How are other platforms supposed to check the super key modifier?

Maybe `modifiers.mac_cmd` can be removed and there should just be `modifiers.command` that maps to the "super" key?

(not strictly related to this PR though - it just stands out)

Reply:
The rational for the unified `command` key is so that you can write something that e.g. selects all with `cmd+A`, and it will use `ctrl+A` on Windows and `⌘+A` on Mac. Changing this behavior is way outside the scope of this PR.

Reply:
yeah I can see the utility of that - though it does still feel odd that there isn't a platform-agnostic modifier for the super/meta/windows/cmd key which is a standard modifier key in its own right. But yep, not related to this PR.

Reply:
I added a note about it at https://github.com/emilk/egui/issues/3653